DigitalLearningSolutions.Web/Styles/claimAccount/completeRegistration.scss

```diff
@@ -0,0 +1,6 @@
+﻿.create-new-login__button {

```

Comment:
Because I want the `<vc:action-link />` (which is an anchor element) to POST the form, I don't know how to do that without having a form with a hidden submit button.

Reply:
Ach... Hmm. Yeah, we shouldn't really be using an action link for a post technically. I'd like to keep the two-action-link feel of this page so that users aren't encouraged towards the "create new login" branch.
So let's add in a confirmation-check page for that branch.

Reply:
I've updated the ticket.